src/index.html

```diff
@@ -28,12 +28,15 @@
 			// set to true if you want to enable comscore tracking
 			comscore: false
 		},
-		autoplay: {
+		// set to true if you want video to autostart
+		autoplay: true,
+		settings: {
 			// set to true when you want to give user option to enable/disable autoplay
 			// autoplay toggle appears in settings menu which sends event `autoplayToggle` on click
 			showToggle: true,
```

Comment:
Can we rename it to `showAutoplayToggle`? It's a little misleading right now, since we moved autoplay outside.